src/modules/library/containers/LibraryContainer.js

```diff
@@ -91,8 +93,8 @@ function LibraryContainer(props) {
 			}
 		} catch (e) {
 			// If response.status !== 2xx then catch block will be executed
-			console.error("Error when retrieving data: ", e); // log the error to the browser's console
-			App.addAlert("warning", t('Unable to load the data', { error: e.toString() }));
+			console.error("Error when retrieving tree data\n", e); // log the error to the browser's console
+			App.addAlert("warning", t('Unable to load the data'));
```

Comment:
@titkov49 I bumped into this generic message several times, maybe it would be a good idea to make it more specific e.g. for tree menu or library (ASABLibraryModule|Unable to load the library content)